cmd/controller/main.go

```diff
@@ -64,6 +64,7 @@ func main() {
 			op.SubnetProvider,
 			op.SecurityGroupProvider,
 			op.InstanceProfileProvider,
+			op.InstanceProvider,
```

Comment:
Was this erroneously removed?

Reply:
Nope! This function changed when we added the tagging controller since that is the only controller that uses the instance provider